staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2269,3 +2261,299 @@ func TestLoadObjects(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyObjects(t *testing.T) {
```

Comment:
nit: let's name this something more applyset-pruning-specific

Reply:
Good idea - done!